community/broot/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+cargo build --release --locked
+
+install -Dm755 "target/release/broot" "$1/usr/bin/broot"
+
+# Build and install manpages.
+release_date=$(sed -n "s/.*v$2 - \(.*\)$/\1/p" CHANGELOG.md)
+cp man/page target/release/broot.1
+sed -i "s/#version/$2/g"         target/release/broot.1
+sed -i "s/#date/$release_date/g" target/release/broot.1
+install -Dm644 target/release/broot.1 "$1/usr/share/man/man1/broot.1"
+
+# Locate and install completions.
+out_dir=$(find target/release/build -type f -name broot.bash -exec dirname {} \; -quit)
```

Comment:
`-quit` is not POSIX, and it's not really necessary here

Reply:
Yeah, that is leftover from the previous approach. FWIW both [GNU](https://manned.org/find/40124615) & [BSD find](https://www.freebsd.org/cgi/man.cgi?query=find&apropos=0&sektion=0&manpath=FreeBSD+12.2-RELEASE+and+Ports&arch=default&format=html) support `-quit`.

Reply:
`toybox` find doesn't :'(

Reply:
Toybox? Kiss's find is BusyBox's find?
```
~ $ find --help 2>&1 | grep quit
	-quit		Exit
```

Reply:
My point is just that we should aim to do the minimum to carry across common implementations KISS users have - I know at least one person using toybox (it's me).

Reply:
Sure, sure, I was just curious how we ended up talking about Toybox all of a sudden. Fully agree and already addressed it...